### PR TITLE
Test gear page evg

### PR DESCRIPTION
**Pull request by evg-qa**

### CODE REVIEW

**Comment by DenisProkofyev** [comment]
Guys, it would better to attach to PRs Trello cards devoted to these TCs. I see the code but I can not see to what cards it is devoted.

For instance, you can use GitHub Power up in Trello or paste links manually in PR

File: tests/gear_page/test_gear_page_evg.py
Comment (gomboc665) [inline tests/gear_page/test_gear_page_evg.py:1]: looks like you are comparing the previous page and the next. but you need to get the current url after the redirection

**Comment by evg-qa** [comment]
https://trello.com/c/RjG4Sdhc/171-tc009005001-gear-page-categories-verify-opening-the-sprite-yoga-companion-kit-page

File: tests/gear_page/test_gear_page_evg.py
Comment (evg-qa) [reply tests/gear_page/test_gear_page_evg.py:1]: I need to find an item and click on it and see that the product page opens, right?

Comment (gomboc665) [reply tests/gear_page/test_gear_page_evg.py:1]: yes, but in your step - you took GEAR PAGE (url) and tried to compare it with the next URL, but you need to click - and get the url of the page after redirection.

Comment (evg-qa) [reply tests/gear_page/test_gear_page_evg.py:1]: review, please

**Review by gomboc665 — APPROVED** [review]
approved

File: tests/gear_page/test_gear_page_evg.py
Comment (gomboc665) [reply tests/gear_page/test_gear_page_evg.py:1]: только мне кажется нужно удалить не твои коммиты, чтоб подтянулись только два твоих обновления (url и тест).

**Pull request merged**